pypxe/dhcp.py

```diff
@@ -122,7 +122,9 @@ def __init__(self, **server_settings):
                     import_safe[packed_mac] = imported[lease]
                 self.leases.update(import_safe)
                 self.logger.info('Loaded leases from {0}'.format(self.save_leases_file))
-            except IOError, ValueError:
```

Comment:
Technically I should have made those proper functions but this gets by the Python3 test for exception changes.